### PR TITLE
Add the url command

### PR DESCRIPTION
**Pull request by rawsyntax**
```
uhc config get url
uhc config set url https://api.openshift.com

### CODE REVIEW

**Review by jhernand — COMMENTED** [review]
I like this idea, but I would like to implement it in a more generic way, so that we can use it for other configuration settings as well. For example, what if I want to get the `token_url` instead of the `url`? Can we make this a generic `config` command with a generic `get` command? Something like this:

```
$ uhc config get url
```

No need to implement all the options right now, with `url` would be enough, but that would give us room to add other settings in the future without having to break the syntax of the command.

**Comment by rawsyntax** [comment]
@jhernand great suggestions -- I will update this PR

**Comment by rawsyntax** [comment]
@jhernand please review. I updated to include `uhc config get` and `uhc config set` for all configurations in `~/.uhc.json`

File: cmd/uhc/config/cmd.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	"github.com/openshift-online/uhc-cli/cmd/uhc/config/set"
+)
+
+var args struct {
+	debug bool
+}
+
+var Cmd = &cobra.Command{
+	Use:   "config COMMAND VARIABLE",
+	Short: "get or set about configuration variables",
```
Comment (jhernand) [inline cmd/uhc/config/cmd.go]: I don't understand very well this short description, specially the word _about_. Maybe it is a leftover?

File: cmd/uhc/config/cmd.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	Run:   run,
```
Comment (jhernand) [inline cmd/uhc/config/cmd.go]: Not completely sure, but I think that for this intermediate command we don't need the `run` function at all. Can you check and if so remove it, please?

File: cmd/uhc/config/get/get.go [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't load config file: %v\n", err)
+		os.Exit(1)
+	}
+	if cfg == nil {
+		fmt.Fprintf(os.Stderr, "Not logged in, run the 'login' command\n")
+		os.Exit(1)
+	}
+
+	switch setting := argv[0]; setting {
```
Comment (jhernand) [inline cmd/uhc/config/get/get.go]: Looks like the `setting` variable isn't really used. Consider removing it.

File: cmd/uhc/config/get/url/url.go [hunk 4]
```diff
@@ -0,0 +1,60 @@
+/*
```
Comment (jhernand) [inline cmd/uhc/config/get/url/url.go]: I think you don't want to add this file, right?

File: cmd/uhc/config/set/set.go [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+		fmt.Fprintf(os.Stderr, "Setting scopes is unsupported")
+		os.Exit(1)
+	case "token":
+		cfg.Token = value
+	case "token_url":
+		cfg.TokenURL = value
+	case "url":
+		cfg.URL = value
+	default:
+		fmt.Fprintf(os.Stderr, "Unknown setting")
```
Comment (jhernand) [inline cmd/uhc/config/set/set.go:93]: And then `os.Exit(1)`.

File: cmd/uhc/config/set/set.go [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+	case "access_token":
+		cfg.AccessToken = value
+	case "client_id":
+		cfg.ClientID = value
+	case "client_secret":
+		cfg.ClientSecret = value
+	case "insecure":
+		cfg.Insecure, err = strconv.ParseBool(value)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to set insecure: %v", value)
```
Comment (jhernand) [inline cmd/uhc/config/set/set.go:76]: And then `os.Exit(1)`.

File: cmd/uhc/main.go [hunk 7]
```diff
@@ -27,6 +27,7 @@ import (
 
 	"github.com/openshift-online/uhc-cli/cmd/uhc/cluster"
 	"github.com/openshift-online/uhc-cli/cmd/uhc/completion"
+	"github.com/openshift-online/uhc-cli/cmd/uhc/config"
```
Comment (jhernand) [inline cmd/uhc/main.go:30]: You are importing `config` but the name of the package is `configcmd`. Either change the name of the package or change this import.

**Review by jhernand — COMMENTED** [review]
Thanks @rawsyntax, looks good. I have some minor comments. Once you adress them I will merge.

**Comment by rawsyntax** [comment]
@jhernand I addressed your comments. Thanks for the review

**Review by jhernand — APPROVED** [review]
Looks good, thanks @rawsyntax.

**Pull request merged**